app/components/raw-config.js

```diff
@@ -0,0 +1,20 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  copied: false,
+
+  buttonLabel: computed('copied', function () {
+    let copied = this.get('copied');
+    if (copied) {
+      return 'Copied!';
+    }
+    return 'Copy to clipboard';
+  }),
+
+  actions: {
+    copied() {
+      this.set('copied', true);
```

Comment:
I would suggest to revert it back to `false` after a few seconds:
```js
this.set('copied', true);
later(() => this.set('copied', false), 3000);
```
Here's the [later](https://api.emberjs.com/ember/3.9/functions/@ember%2Frunloop/later) docs

Reply:
Old moved code, but agree its worth to refactor